vsintegration/src/FSharp.Editor/Completion/CompletionService.fs

```diff
@@ -43,6 +46,41 @@ type internal FSharpCompletionService
             .WithDismissIfLastCharacterDeleted(true)
             .WithDefaultEnterKeyRule(enterKeyRule)
 
+    override _.GetDefaultCompletionListSpan( sourceText , caretIndex ) =
+
+        let mutable startIndex = 0
+        let mutable endIndex = 0
+
+        // Get single line text and index
+        let textLines = sourceText.Lines
+        let lineText = textLines.GetLineFromPosition(caretIndex).ToString()
+        let lineCaretIndex = textLines.GetLinePosition(caretIndex).Character
+
+        // Check for enclosing backticks or leading backticks, else capture valid identifier characters
+        // TODO Replace naive backtick check with safer alternative
+        match lineText.IndexOf "``", lineText.LastIndexOf "``" with
```

Comment:
This is actually likely to be safe. Perhaps pulling this function out to check would be better? https://github.com/dotnet/fsharp/blob/main/vsintegration/src/FSharp.Editor/LanguageService/Tokenizer.fs#L811